src/bindings/python/src/openvino/frontend/tensorflow/utils.py

```diff
@@ -300,7 +309,7 @@ def create_tf_graph_iterator(input_model, placeholder_shapes, placeholder_data_t
         if hasattr(input_model, 'inputs') and hasattr(input_model, 'structured_input_signature'):
             internal_tensor_names = []
             for func_input in input_model.inputs:
-                if func_input.dtype == tf.resource:
+                if is_variable(func_input, input_model.graph.captures):
```

Comment:
what is captures? are we sure that it always exists?
`https://www.tensorflow.org/api_docs/python/tf/Graph`

Reply:
why are these changes needed?

Reply:
Changed this part as discussed offline.